storage_service/administration/views.py

```diff
@@ -76,3 +77,24 @@ def user_create(request):
             username=user_form.cleaned_data['username']))
         return redirect('user_list')
     return render(request, 'administration/user_form.html', locals())
+
+
+########################## LANGUAGE ##########################
+
+def change_language(request):
+    current_language = get_language()
+    language_selection = None
+
+    # Fall back to a generic language if the selected language
+    # is too specific (e.g. en-us can fall back to en)
```

Comment:
Did you have to do this because `DEFAULT_LANGUAGE` defaults to `en-us`? I think it's ok to change it to `en`. For some reason I didn't run into any issues related to this in the Dashboard.